lambdas/build-cri-oauth-request/src/main/java/uk/gov/di/ipv/core/buildcrioauthrequest/BuildCriOauthRequestHandler.java

```diff
@@ -402,7 +402,7 @@ private SharedClaimsResponse getSharedAttributesForUser(
                     SharedClaims credentialsSharedClaims =
                             OBJECT_MAPPER.readValue(
                                     credentialSubject.toString(), SharedClaims.class);
-                    if (credentialIss.equals(configService.getComponentId(ADDRESS))) {
+                    if (ADDRESS.equals(vc.getCri())) {
```

Comment:
(this is a subtle difference, and what caused the tests to break - the issuer was not set correctly in the test VCs, even though the tests notionally used address VCs)